docker/update.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ -v PIZZA_UP ]
+then
+	# update repo
+	cd /opt/pizza-sync
+	git pull --rebase
```

Comment:
I don't think it should be the responsibility of the image to git pull stuffs…

For me, once the image is built, everything stay frozen except for data

so you do the yarn calls and build during build, not during execution

Reply:
Idea here is to be able to update the code at "docker create" time, without rebuilding the image.

Reply:
I know: I think it is a bad idea :)